modules/500-operator-trivy/templates/pdb.yaml

```diff
@@ -0,0 +1,15 @@
+---
+apiVersion: policy/v1
+kind: PodDisruptionBudget
+metadata:
+  annotations:
+    helm.sh/hook: post-upgrade, post-install
```

Comment:
do we really need the hook annotation here? PDB works well without it but this annotation can create some stupid problems on update

Reply:
Honestly, I've just copied it from another PDB in the codebase. I've assumed that it's some forbidden magical incantation to please Helm.

Reply:
Fixed.